src/components/image/image.tsx

```diff
@@ -85,7 +85,7 @@ type _EuiImageSrcOrUrl = ExclusiveUnion<
 
 export type EuiImageProps = CommonProps &
   _EuiImageSrcOrUrl &
-  HTMLAttributes<HTMLImageElement> & {
+  ImgHTMLAttributes<HTMLImageElement> & {
```

Comment:
```suggestion
  Omit<ImgHTMLAttributes<HTMLImageElement>, 'src' | 'alt'> & {
```

This should get the errors resolved, @lildeadprince